SELearning.API/Controllers/CommentController.cs

```diff
@@ -19,21 +20,19 @@ public CommentController(ILogger<CommentController> logger, ICommentRepository r
         _repository = repository;
     }
 
-    [Authorize]
     [HttpGet("{ID}")]
     [ProducesResponseType(typeof(CommentDTO), 200)] // OK
     [ProducesResponseType(404)] // Not Found
     public async Task<ActionResult<CommentDTO>> GetComment(int id)
         => (await _repository.GetAsync(id)).ToActionResult();
 
-    [Authorize]
     [HttpGet("{contentID}")]
     [ProducesResponseType(typeof(CommentDTO), 200)] // OK
     [ProducesResponseType(404)] // Not Found
     public async Task<IReadOnlyCollection<CommentDTO>> GetCommentsByContentID(int contentID)
         => await _repository.GetAsyncByContentID(contentID);
 
-    [Authorize]
+    [AuthorizePermission(Permission.CreateComment)] // TODO: Create the possibility to have an 'or' evaluation of rules in the permission attribute and policy provider.
```

Comment:
TODO's in master?

Reply:
Jada, de er også i vores Blazor-filer :D

Reply:
Altså en TODO er do later, ikke do now ;))))

Reply:
Er altid bange for TODO's. I mit hoved ender de med at blive til "DON'T DO"

Reply:
Vi fikser dem på en anden branch om lidt